### ISSUE TITLE
Error on install

### ISSUE THREAD

**Issue by ipatate** [jamesedmonston/graphql-authentication#10]
When i try to install the plugin directly from the store, i have this error : 

```
ParseError: syntax error, unexpected ')'
```

Craft version : Craft Pro 3.5.15.1

![Capture d’écran 2020-11-13 à 09 20 34](https://user-images.githubusercontent.com/46760/99045531-c9240a00-2591-11eb-8b80-39b0cecbf711.png)

**Comment by jamesedmonston** [comment]
@ipatate thanks for reporting this. Looking into it now.

**Comment by jamesedmonston** [comment]
@ipatate hmm, I can't seem to replicate this issue. Do you have any custom modules running that could have a syntax error in them?

Would you be able to paste in your `phperrors.log` file from `/storage/logs/`, please?

**Comment by ipatate** [comment]
No module and just 2 plugins (redactor and webhook)
Sorry, i don't  have file ```phperrors.log``` in ```/storage/logs/```.
But i have more info for this error :
```
ParseError: syntax error, unexpected ')' in /var/www/craft35/vendor/jamesedmonston/graphql-authentication/src/GraphqlAuthentication.php:108
```
I don't see the problem in this line !

**Comment by jamesedmonston** [comment]
@ipatate are you running a PHP version earlier than 7.3? Looks like it could be to do with the trailing commas. I'll push a release that removes those

**Comment by ipatate** [comment]
yes it's PHP 7.2

**Comment by jamesedmonston** [comment]
@ipatate I've just released [`1.1.7`](https://github.com/jamesedmonston/graphql-authentication/releases/tag/1.1.7) which should (hopefully) resolve this for you

**Comment by ipatate** [comment]
it's work now, thank you 🥇
